drivers/regulator/pmic_shell.c

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2022 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/zephyr.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/drivers/uart.h>
+#include <zephyr/drivers/regulator.h>
+#include <zephyr/drivers/regulator/consumer.h>
+
+LOG_MODULE_REGISTER(regulator_shell, CONFIG_REGULATOR_LOG_LEVEL);
+
+#define GET_REGULATOR(node) DEVICE_DT_GET(node),
```

Comment:
What's the point here? You're just making an alias for an established, well-known macro?

Reply:
yeah, this is wrong- I've removed it